scripts/azurepowershellscript.ps1

```diff
@@ -0,0 +1,20 @@
+$resourceGroupName="myResourceGroup"
+$location = "East US"
+$ehubName="myEventHub"
+$namespaceName="myNamespace2023"
+$functionAppName = "AzureFunctionTriggerApp2023"
+$runtime = "powershell"
+$storageAccountName = "testdatastorage2023"
+$accountName = "mydb2023"

```

Comment:
Name is not appropriate

Reply:
We tried naming standards but it was not accepting names we were giving as they were already taken. So we had to go for these after trying multiple times with multiple names.